maquetado/index.html

```diff
@@ -6,9 +6,22 @@
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
+	<link rel="stylesheet"  type="text/css" href="application.css">
+	<link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
 	<h2 class="title">Front-End Tools</h2>
+	<div class="technology">
+		<div class="border-ext">
```

Comment:
You dont need this div and the one inside, you can use a pseudo element in `technology` div

Reply:
Also, check the indent here. You have a lot of white space, we usually use just 2